internal/store/store.go

```diff
@@ -101,6 +129,16 @@ func NewBlockStore(db dbm.DB, options ...BlockStoreOption) *BlockStore {
 	return bStore
 }
 
+func (bs *BlockStore) GetVersion() string {
+	switch bs.dbKeyLayout.(type) {
+	case *v1LegacyLayout:
+		return "v1"
+	case *v2Layout:
+		return "v2"
+	}
+	return "no version set"
```

Comment:
Should we default to `v1`?

Reply:
I don't think so because at this point in the code we should have set this. So if this return false it is a scenario not supported by us. It is true someone else might have called it before but I think it's better to return this rather than assume something.